Quickerfix

TadJohnson00 opened the pull request:
Replaces the vast majority of armour in organic play with Delta V variants which: A) look nicer, B) are specialised, C) are less cringe B-)

<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
Added new armour types that replace most of the BasicArmour and BasicArmourSlim instances in organic play; barkeeps now have stabproof vests instead of generic body armour, security now has the option of damage resist types (see below). Most of the stuff should be adequately commented too :)
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
Having bullet-resistant and stab-resistant vests helps give security the option of bullet-resistance and lower speed, or stab- and blunt-resistance at full speed. Plus the sprites and balance needed to be reworked anyways

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
N/A
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->
![image](https://github.com/DeltaV-Station/Delta-v-rebase/assets/120885811/d72dd082-2a26-4b11-833b-270d579c738d)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->
:cl: TJohnson
- add: Added new vests to play to replace most of the Type I vests in organic play.
- remove: Removed most of the Type I vests in organic play. Some might still crop up, but don't count on it!

DebugOk reviewed (changes requested):
You should savegrid instead of savemap

Carolyn3114 commented:
> You should savegrid instead of savemap

shit how do I change it

DebugOk commented:
Merging from master which caused another commit to slip in, please remake this pr from a feature branch

TadJohnson00 committed:
Plate Carriers and Stabproof Vests

Replaces the vast majority of armour in organic play with Delta V variants which: A) look nicer, B) are specialised, C) are less cringe B-)

TadJohnson00 committed:
Adds Resistances to Winter Coats

Captain's winter coat and security winter coat are updated, greatcoat updated to match sec winter coat

Colin-Tel reviewed (approved):
🥴
